src/lib/components/streams/create-channel-modal.svelte

```diff
@@ -117,6 +124,11 @@
                     <option value={ChannelType.private}>Private Channel</option>
                     <option value={ChannelType.public}>Public Channel</option>
                 </Input>
+                {#if channelType === ChannelType.private}
+                    <Label class="mt-3">Use preshared key</Label>
```

Comment:
this should be in one line like the hidden checkbox but using a switch is good 👍  and please remove mt-3